scripts/review_comments_retriver.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
```

Comment:
Typo in the name of the file :)

Reply:
Fixed in d227893d7d02c4b9f1d0c1728741190e99b763d8.